### ISSUE TITLE
Confirm sign up component not customizible

### ISSUE THREAD

**Issue by adampblack** [aws-amplify/amplify-js#7184]
**Describe the bug**
Trying to customise the AmplifyConfirmSignUp and it is not overriding the default.

**To Reproduce**
<AmplifyConfirmSignUp
          headerText={t('Verify your mobile phone number')}
          slot="confirm-sign-in"
          submitButtonText={t('Confirm')}
  />

**Expected behavior**
Expect the header text and submitbuttontext to change with whatever I set.

**Comment by lukasrossa** [comment]
Are you sure the confirm-sign-in slot is valid for the AmplifyConfirmSignUp component? Shouldn't it be slot="confirm-sign-up" ?

**Comment by adampblack** [comment]
Yes, you are correct.

I have just tested this and the slot setting was the issue.


The root cause of this problem was that the AmplifyConfirmSignIn code was shown under the AmplifyConfirmSignUp component which I had copied and pasted - I changed the component name but forgot to change the slot setting.
https://github.com/aws-amplify/docs/issues/2659#issuecomment-728105325

Thanks for suggestion, much appreciated.
